phpthread.go

```diff
@@ -73,11 +73,10 @@ func (thread *phpThread) getActiveRequest() *http.Request {
 // Pin a string that is not null-terminated
 // PHP's zend_string may contain null-bytes
 func (thread *phpThread) pinString(s string) *C.char {
-	if s == "" {
```

Comment:
I'm not sure of the benefit of this change. Could you explain the rationale a bit more?